platform-common-date/src/test/java/com/softicar/platform/common/date/DayTest.java

```diff
@@ -251,4 +252,56 @@ public void testHashCode() {
 
 		assertEquals(Day.fromYMD(2015, 9, 28).hashCode(), Day.fromYMD(2015, 9, 28).hashCode());
 	}
+
+	@Test
+	public void testFromYMDChecked() {
```

Comment:
A corresponding `testFromYMD()` appears to be missing... we kind of lack some symmetry here, at least.

Reply:
Hmm, I did not change anything on the `fromYMD()` because that was never my aim...I rather thought about deprecating that method...on the other hand there might be some proper use cases for it, so i am undecided.

Reply:
TBH, I see no point in retaining the gracefully-wrapping `Day#fromYMD` methods, considering that we have `Day#getRelative(int)`. The behavior has always been kind of magic. So, IMO, deprecating those methods would indeed be preferrable.

Reply:
decided to add a test